data/json/items/generic.json

```
@@ -592,6 +592,20 @@
     "pocket_data": [ { "pocket_type": "SOFTWARE", "max_contains_volume": "1 L", "max_contains_weight": "1 kg" } ],
     "to_hit": -3
   },
+  {
+    "type": "GENERIC",
+    "id": "fema_data",
+    "symbol": ",",
+    "color": "white",
+    "name": { "str": "FEMA data" },
```

Mark plural form to be `FEMA data` too, rather than `FEMA datas`:
```suggestion
    "name": { "str_sp": "FEMA data" },
```